packages/plans-grid/src/titles/style.scss

```diff
@@ -0,0 +1,15 @@
+@import 'assets/stylesheets/gutenberg-base-styles';
+@import '../mixins';
+
+.plans-ui-title {
+	@include onboarding-heading-text-mobile;
+	color: var( --mainColor );
```

Comment:
Ahh, carrying these color vars will be a challenge.

I think we should drop them already now and rely on regular `--studio-*` colors instead.